syne_tune/optimizer/schedulers/searchers/kde/kde_searcher.py

```diff
@@ -95,13 +93,13 @@ def __init__(
         config_space: dict,
         metric: str,
         points_to_evaluate: Optional[List[dict]] = None,
-        mode: str = "min",
-        num_min_data_points: int = None,
-        top_n_percent: int = 15,
-        min_bandwidth: float = 1e-3,
-        num_candidates: int = 64,
-        bandwidth_factor: int = 3,
-        random_fraction: float = 0.33,
```

Comment:
Same as syne_tune/optimizer/schedulers/searchers/bore/bore.py - I think this is more confusing now - users wont know the default values just by looking at the API but will need to dive into the implementation.

Reply:
NOTE: as long as the default values are in the docstring it makes sense to me.